src/all.css

```diff
@@ -0,0 +1,2 @@
+@import './styles/index.css';
+@import './lib/index.css';
```

Comment:
Can we just compile all the CSS via the webpack config, rather than having to create an `index.css` for every component?